user.js

```diff
@@ -661,13 +675,49 @@ module.exports = function user (options) {
 
       login.save$(function (err, login) {
         if (err) return done(err)
-
-        seneca.log.debug('login/ok', why, user, login)
-        done(null, {ok: true, user: user, login: login, why: why})
+        seneca.act({role: role, cmd: 'set_user_lock', id: user.id}, function (err, out) {
+          if (err) return done(err)
+          seneca.log.debug('login/ok', why, user, login)
+          done(null, {ok: true, user: user, login: login, why: why})
+        })
       })
     }
   }
 
+  // Unlocks accounts that a user has been locked out of due to failled password attempts
+  // Or let the lock if the user failed to login to their account
+  // - id, user id to unlock
+  // - failTry, boolean to say login failed
+  // - why, why account lock was changed
+  // Provides:
+  // - success: {ok:true, why:}
+  // - failure: {ok:false, why:}
+  function cmd_set_user_lock (args, done) {
+    var seneca = this
+    var why
+    var userent = seneca.make(user_canon)
+    var ok
+
+    userent.load$({id: args.id}, function (err, user) {
+      if (err) return done(err)
```

Comment:
@Wardormeur well here the coding standard requires one var/line so its OK like it is now.